pyrex.ini

```diff
@@ -103,6 +103,7 @@ envvars =
 #   --device /dev/kvm
 args =
 	--device /dev/kvm
```

Comment:
@amstewart Why do we need this?

Reply:
The [build.vms.sh](https://github.com/ni/nilrt/blob/nilrt/master/hardknott/scripts/pipelines/build.vms.sh) pipeline script is [run within the pyrex container](https://github.com/ni/nilrt/blob/nilrt/master/hardknott/scripts/pipelines/build.vms.sh#L207) and uses QEMU/KVM to create the NILRT VM artifacts. The `--device /dev/kvm` arg just enables the pyrex container to use the container-host's KVM device, if present.

Reply:
Ok, so why didn't we add it to azdo pyrex.ini also?

Reply:
Is kvm device not available on azdo agents?

Reply:
It might not have been. I think I attempted to add it to both when I originally authored this stuff, but that was over a year ago. The host configs may have changed.

Reply:
Seems like its available now. I'll make that a separate change once I test it.